circuits/cpp/src/aztec3/circuits/kernel/private/common.cpp

```diff
@@ -67,6 +124,9 @@ void common_update_end_values(DummyComposer& composer,
 
     const auto& storage_contract_address = private_call_public_inputs.call_context.storage_contract_address;
 
+    // TODO(dbanks12): might need to silo read requests so that they match new commitments!!!!
```

Comment:
New commitments are commitments to the newly formed notes as a part of a transaction. Read requests are commitments of notes that are already present in the private data tree. How are they related?

Reply:
I think you are right, this is just not necessary.

But just to clarify, read requests are unsiloed commitments of notes. In order to do a membership check on a read request, it must first be siloed by storage contract address. This is what is done here https://github.com/AztecProtocol/aztec-packages/pull/619/files#diff-e0ce18b9a7595f0bc350f37627b815fc97c5e9ad9fa8dd909ef42930ad42130aR86 when read requests are validated.